R/Rules-class.R

```diff
@@ -1960,7 +1960,7 @@ setClass(
 #' @rdname CohortSizeRange-class
 #'
 #' @param intervals (`numeric`)\cr see slot definition.
-#' @param cohort_size (`integer`)\cr see slot definition.
+#' @param cohort_size (`numeric`)\cr see slot definition.
```

Comment:
but it cannot be any numeric, but one that is an integer only? ... see slot definition. or did we always do like this because the constructor takes numerics that can be coerced to integer too?